src/codecovio.jl

```diff
@@ -182,27 +182,17 @@ module Codecov
     """
     function submit_generic(fcs::Vector{FileCoverage}; kwargs...)
         @assert length(kwargs) > 0
-        dry_run = false
-        verbose = true
```

Comment:
We should implement an informative deprecation, so that people using the `verbose` keyword know that to get the same logging level, they should use the logging framework instead. I'd add something like
```julia
if haskey(kwargs, :verbose)
    Base.depwarn("The verbose keyword argument is deprecated, set the environment variable " *
                 "JULIA_DEBUG=Coverage for verbose output", :submit_generic)
    verbose = kwargs[:verbose]
else
    verbose = false
end
```
then keep all existing logic in place for the duration of the deprecation period. That avoids a hard break for existing code.

Reply:
Are you aware of any code or people using the "verbose" mechanism?

Reply:
(Just to clarify, I'll do what you suggest anyway, though, but am still wondering)

Reply:
I don't, it's more just a precaution.

Reply:
@ararslan OK now?